src/test/java/com/arpnetworking/metrics/mad/parsers/JsonToRecordParserV2cTest.java

```diff
@@ -47,9 +47,13 @@ public void testParse() throws ParsingException, IOException {
         final Record record = parseRecord("QueryLogParserV2cTest/testParse.json");
 
         Assert.assertNotNull(record);
-        Assert.assertEquals("MyCluster", record.getAnnotations().get(Key.CLUSTER_DIMENSION_KEY));
-        Assert.assertEquals("MyService", record.getAnnotations().get(Key.SERVICE_DIMENSION_KEY));
-        Assert.assertEquals("MyHost", record.getAnnotations().get(Key.HOST_DIMENSION_KEY));
+
```

Comment:
Since you've moved all the annotations to dimensions you should add test cases to ensure that non-dimension annotations continue to be parsed correctly. This goes for the other formats as well.